Deep-Spread.md

```diff
@@ -0,0 +1,92 @@
+All credit for this idea goes to Lee Byron @leebyron!
+
+The additional syntax below can be introduced as a follow up proposal.
+
+# Deep Spread for Records
```

Comment:
I wouldn't call this "deep spreads", more like, "deep property definitions". It's just used *in conjunction with* spread.